### ISSUE TITLE
Black titlebar  on Chrome

### ISSUE THREAD

**Issue by woomia** [nick-gravgaard/windowlab#5]
When I use Chrome on Windowlab, the titlebar is black. Anyone have an ideas?

**Comment by nick-gravgaard** [comment]
I think Chrome has a setting which tells it to decorate its window itself. Have you tried changing that?

**Comment by woomia** [comment]
Yes, it shows window decor of some kind but the titlebar above it is still black.

**Comment by nick-gravgaard** [comment]
Can you take a screenshot?

**Comment by woomia** [comment]
![image](https://user-images.githubusercontent.com/1365979/152077163-b212abeb-f50e-47ed-883b-7543113c2a20.png)

**Comment by nick-gravgaard** [comment]
Thanks. I've not heard of this problem before, but I don't use or develop WindowLab anymore. It would be interesting to know what the first version of Chrome was that had this issue. Do any other recently compiled programs have this problem? Recent GNOME programs for instance?

**Comment by narinishi** [comment]
The same thing happens with Firefox. Manually enabling the title bar from the customization settings fixes it.

In Chrome, if you right-click on an empty space in the tab bar, there should be an option to "Use system title bar and borders". I think that should solve the issue in Chrome.
